libs/positions/tailwind.config.js

```diff
@@ -0,0 +1,17 @@
+const { join } = require('path');
```

Comment:
question (non blocking): I didn't think libs needed their owntailwind config? Is this required to get styles for storybook working?

Reply:
Tailwind config is only for storybook.